src/main/kotlin/com/exactpro/th2/common/message/MessageFilterUtils.kt

```diff
@@ -129,6 +134,17 @@ private fun RootComparisonSettings.toTreeTableEntry(): TreeTableEntry = Collecti
                 .build())
         }
     }.build())
+    if (hasTimePrecision()) {
+        val formattedDuration = DurationFormatUtils.formatDuration(Durations.toMillis(timePrecision), DEFAULT_TIME_PRECISION_FORMAT)
+        row("time-precision", RowBuilder()
+            .column(IgnoreFieldColumn(DEFAULT_TIME_PRECISION_REGEX.replace(formattedDuration, "").trim()))
```

Comment:
If you are willing to use regexp maybe it is better to use something like that instead of DurationFormatUtils
https://stackoverflow.com/a/40487511